api/views.py

```diff
@@ -151,16 +152,16 @@ def sendText(numbers):
                     body= f'\nToday\'s NASA Astronomy Picture of the Day is: {data["title"]}.\n\n{data["explanation"]}',
                     media_url=data['hdurl']
                 )
-                print(message.sid)
+                return message.sid
             else:
                 message = client.messages.create(
                 to=number,
                 from_=os.environ.get('TWILIO_PHONE_NUMBER'),
                 body= f'\nToday\'s NASA Astronomy Picture of the Day is: {data["title"]}.\n\n{data["explanation"]}'
                 )
-                print(message.sid)
-        except TwilioRestException as e:
-            print(e)
+                return message.sid
```

Comment:
this return value doesn't seem to be used when called from `sendAPOD`, do you plan on doing something with it?

Reply:
for this, I will just print to the console but, I want to look into logging, what do you think?

Reply:
console log is fine, a logging framework would be overkill